docs/source/wakepy-mode-lifecycle.md

```
@@ -141,13 +166,14 @@ long_running_task() # activates and deactivates
 
 ```
 
-The python process creates a new Mode context manager instance and uses it every time a decorated function is called. If the process enters a *Activation Failed* state, the *Action on Fail* occurs (See: {numref}`fig-mode-activity-diagram`). This action may be an exception, a warning, or a custom action as determined by the [`on_fail`](#wakepy.keep.running) input parameter.
+The python process creates a new Mode context manager instance and uses it every time a decorated function is called. If the process enters a *Activation Failed* state, the *Action on Fail* occurs (See: {numref}`fig-mode-activity-diagram`). This action may be an exception, a warning, or a custom action as determined by the {func}`on_fail <wakepy.keep.running>` input parameter.
 
 
 The {numref}`fig-activate-mode-activity-diagram` presents an activity diagram from the "Activate Mode" step of {numref}`fig-mode-activity-diagram`. The steps are:
 - ***Check WAKEPY_FAKE_SUCCESS***: If the [`WAKEPY_FAKE_SUCCESS`](#WAKEPY_FAKE_SUCCESS) environment variable is set to a truthy value, the `WakepyFakeSuccess` method is inserted at the beginning of the methods list. This special method has `.caniuse()`, `.enter_mode()`, `.heartbeat()` and `.exit_mode()` which always succeed without making any real system calls, which is useful for testing.
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)